portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/FormsTranslations.testcase

```diff
@@ -888,14 +888,54 @@ definition {
 	}
 
 	@description = "Verify that a Paragraph field's Label and Body Text can be translated"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test ParagraphFieldCanBeTranslated {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133479 ParagraphFieldCanBeTranslated pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Paragraph");
+
+		FormFields.editParagraphBody(content = "Paragraph body text.");
+
+		Form.addTranslation(localizedLanguage = "Spanish (Spain)");
+
+		Form.editFieldLabelMetal(fieldValue = "Párrafo");
+
+		FormFields.editParagraphBody(content = "Texto de Párrafo");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		FormViewBuilder.viewTranslationLanguage(symbolName = "en-US");
+
+		FormFields.viewParagraphField(
+			fieldName = "Paragraph",
+			fieldTitle = "Paragraph",
+			fieldValue = "Paragraph body text.");
+
+		FormViewBuilder.changeAppLanguageTo(changeAppLanguageTo = "en-US");
+
```

Comment:
Hi @lima-rafa. I'm not sure about if this line is necessary since it's not on the [ticket](https://issues.liferay.com/browse/LPS-133479) steps? Please, could you clarify it to me so that I do not question this again in the next reviews? =D In case it's not necessary, you can remove it ok? Thank you! =)

Reply:
Sorry, I've found that this line is really necessary. =)